cfd_protocol/tests/cfds.rs

```diff
@@ -112,20 +132,23 @@ fn renew_cfd() {
     let maker_wallet = build_wallet(&mut rng, Amount::from_btc(0.4).unwrap(), 5).unwrap();
     let taker_wallet = build_wallet(&mut rng, Amount::from_btc(0.4).unwrap(), 5).unwrap();
 
-    let oracle_data = OliviaData::example();
+    let oracle_data = OliviaData::example_0();
     let oracle_pk = oracle_data.pk;
-    let nonce_pks = oracle_data.nonce_pks.clone();
-
-    let payouts = vec![
-        Payout::new(0..=10_000, Amount::from_btc(2.0).unwrap(), Amount::ZERO).unwrap(),
-        Payout::new(
-            10_001..=50_000,
-            Amount::ZERO,
-            Amount::from_btc(2.0).unwrap(),
-        )
-        .unwrap(),
-    ]
-    .concat();
+    let event = oracle_data.announcement();
+
+    let payouts_per_event = HashMap::from_iter([(
+        event,
+        vec![
+            Payout::new(0..=10_000, Amount::from_btc(2.0).unwrap(), Amount::ZERO).unwrap(),
+            Payout::new(
+                10_001..=50_000,
+                Amount::ZERO,
+                Amount::from_btc(2.0).unwrap(),
+            )
+            .unwrap(),
```

Comment:
I find it a bit odd to return `Vec<Self>` from a `new` function but it will do for now :)

Reply:
We can rename it if it helps, but I think the behaviour of generating multiple `Payout`s per interval is still desirable/necessary.